### ISSUE TITLE
Is wa-sqlite-web-backend supported?

### ISSUE THREAD

**Issue by kasual1** [kikko-land/kikko#130]
I'm wondering whether I can use wa-sqlite-web-backend instead of absurd-sql-backend already. I haven't found anything related in the docs. If you need a helping hand, let me know.

Also: what are the current benefits of using wa-sqlite-web-backend? Is there a performance gain?

Thanks for building kikko! Integration with absurd worked very well.

**Comment by quolpr** [comment]
@kasual1 here you can check the difference https://github.com/kikko-land/wa-sqlite-web-backend/issues/35 .

Currently, wa-sqlite backend could not work correctly. I need some time to polish the new backend API, and then I will fix wa-sqlite backend. Or if you need wa-sqlite integration by now, I can fix it 🙂

**Comment by kasual1** [comment]
@quolpr Thanks Sergey for your response!
No there is no need to fix it immediately. 
I can stick with absurd-sql for now and make the switch as soon as you're ready. Due to your implementation it's really easy to switch backends, thanks for that!

I'm rather curious whether using wa-sqlite-web-backend would increase performance of mass insertion, compared to absurd-sql. I'm working on a performance critical app and need to know if the switch to wa-sqlite could improve my current insertion speed.

**Comment by quolpr** [comment]
@kasual1 as for the performance, absurd-sql will always be the fastest because it doesn't use asyncify 🙂

**Comment by stereosteve** [comment]
@quolpr on a related note are there plans to add backend for the official sqlite wasm build?

https://sqlite.org/wasm/doc/trunk/about.md

While OPFS is not yet everywhere... it seems to address most of the concerns from the absurd-sql POC.

**Comment by quolpr** [comment]
@stereosteve good idea, created issue https://github.com/kikko-land/kikko/issues/137

**Comment by quolpr** [comment]
Since wa-sqlite is under MIT now, I moved wa-sqlite to this monorepo & adopted to new backend API. The next step will be to update Doc and deprecate absurd-sql adapter

Here how you can use wa-sqlite:
https://github.com/kikko-land/kikko/blob/main/packages/vite-react-example/src/list/AppList.tsx#L31
